lib/modules/manager/terragrunt/index.ts

```diff
@@ -2,6 +2,7 @@ import { GitTagsDatasource } from '../../datasource/git-tags';
 import { GithubTagsDatasource } from '../../datasource/github-tags';
 import { TerraformModuleDatasource } from '../../datasource/terraform-module';
 
+export { updateArtifacts } from './lockfile';
```

Comment:
please rename, because all other managers are using `artifacts`
```suggestion
export { updateArtifacts } from './artifacts';
```


we should rename terraform manager too in a separate PR.